spring-cloud-starter-app-broker-cloudfoundry/build.gradle

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+description = "Spring Cloud App Broker Starter for Cloud Foundry"
+
+dependencyManagement {
+	imports {
+		mavenBom "org.springframework.boot:spring-boot-dependencies:${springBootVersion}"
+	}
+}
+
+dependencies {
+	api project(":spring-cloud-starter-app-broker")
+	api project(":spring-cloud-app-broker-deployer-cloudfoundry")
+	api project(":spring-cloud-app-broker-security-credhub")
+
+	api("org.cloudfoundry:cloudfoundry-client-reactor:${cfJavaClientVersion}")
+	api("org.cloudfoundry:cloudfoundry-operations:${cfJavaClientVersion}")
+	api("io.projectreactor.ipc:reactor-netty")
+	api("org.immutables:value:${immutablesVersion}")
```

Comment:
I don't think we should include this immutables dependency in the starter. I added it to the other build files to clean up some Gradle warnings, but it should really be handled by Reactor. And maybe it's actually a bug that we have to include it at all?

Reply:
I'll try removing that and seeing what happens with the build and warnings. I'm pretty sure it isn't needed at runtime.